.github/workflows/test.yml

```diff
@@ -74,6 +74,8 @@ jobs:
           gemfile: Gemfile.5.2.pg
         - ruby: 2.5.3
           gemfile: Gemfile.6.0.pg
+        - ruby: 2.5.3
```

Comment:
Can you use Ruby 2.6.6 here? Background: This is the version we currently use most often in our projects and it would be nice to see this is working well for `edge_rider`. I will also create a story to discuss when we want to support 2.7 and 3.x. In 2.7 there were many deprecation warnings in the past, so it requires you to upgrade to the most recent versions of the dependencies in Gemfile to get rid of them + fix the own deprecation warnings.